src/components/rating/pages/rating-interactive.jsx

```diff
@@ -21,15 +21,19 @@ const Ratings = () => {
       values: Number
     },
     {
-      name: 'counter',
-      values: Number
+      name: 'counterText',
```

Comment:
counter text is "votes"?

Reply:
ah, I see below. It's the whole thing "34 votes".

Reply:
yes ' XX votes'